packages/twenty-ui/src/display/avatar/components/Avatar.tsx

```diff
@@ -55,16 +56,20 @@ export const Avatar = ({
   color,
   backgroundColor,
 }: AvatarProps) => {
-  const [isInvalidAvatarUrl, setIsInvalidAvatarUrl] = useState(false);
+  const [invalidAvatarUrls, setInvalidAvatarUrls] = useRecoilState(
```

Comment:
We would like to avoid a solution with recoil because this component is optimized for performance and displayed many times in a table.

Could you instead work on a solution that keeps only a React useState and maybe show a the placeholder while loading the image ?

Only using React states with handlers on the img component seems like a good approach for keeping performance low.

You can test the performance of the component with the command `npx nx storybook:dev twenty-front --configuration=performance` and then run the performance test for a relation field display for example.

It would be a good idea to add a storybook test with a wrong url also !

Reply:
Alright @lucasbordeau 
Let's update the approach!

Reply:
@lucasbordeau Whenever the mouse moves in, `RecordTableCellContainer` updates its `hasSoftFocus` state to true, and also sets it to false when the mouse leaves. This forces it's children to re-render (unmount and mount) forcing their state to be reset.

Haven't found a good way to solve this yet. All in all we need to persist this state between the re-renders. Perhaps we can move the state to a context?!

Reply:
Alright, let's keep it like that, I think the problem should be solved by finding a way to not re-render the display component when hasSoftFocus changes but it's out of the scope of this PR, let's keep this solution for now, even if it's not ideal.